ci-operator/config/openshift/vmware-vsphere-csi-driver-operator/openshift-vmware-vsphere-csi-driver-operator-master.yaml

```diff
@@ -63,6 +63,18 @@ tests:
       TEST_SKIPS: provisioning should mount multiple PV pointing to the same storage
         on the same node
     workflow: openshift-e2e-vsphere
+- as: operator-e2e-test
```

Comment:
looking at the other e2e jobs: `e2e-vsphere`, `e2e-vsphere-csi`, `e2e-vsphere-csi-extended`, `e2e-vsphere-zones`, should it be `e2e-vsphere-operator`?